zappnode-front/src/landing/index.js

```diff
@@ -1,12 +1,17 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
```

Comment:
Try to clean up unused imports if you can with `eslint --fix src/`

```suggestion
import React, { useState} from 'react';
```

Reply:
okay